packages/client/src/views/SearchResult/AdvancedSearch.tsx

```diff
@@ -77,6 +77,19 @@ const dateFieldTypes = [
   'informantDoB'
 ]
 
+function isInvalidDate(date: string) {
+  const regEx = /^\d{4}-\d{2}-\d{2}$/
+  if (!date.match(regEx)) {
+    return true
+  }
+  const d = new Date(date)
+  const dNum = d.getTime()
+  if (!dNum && dNum !== 0) {
+    return true
+  }
+  return d.toISOString().slice(0, 10) !== date
+}
```

Comment:
There must be a function for this in date-fns 🤔

Reply:
Okay. I got this from metrics :grinning: